src/main/java/com/github/underscore/U.java

```diff
@@ -283,6 +283,20 @@ public void forEachRight(final Consumer<? super T> func) {
         eachRight(iterable, func);
     }
 
+    public static <T> T[] fill(T[] array, T item) {
+        for (int i = 0; i < array.length; i++) {
```

Comment:
We need  to create a copy for `array`. The idea to not modify source array, but create a new one.